build.sbt

```diff
@@ -1304,13 +1305,18 @@ lazy val `ydoc-server` = project
   .dependsOn(`logging-service-logback`)
   .dependsOn(`profiling-utils`)
 
+lazy val persistanceVersion = "0.2-SNAPSHOT"
 lazy val `persistance` = (project in file("lib/java/persistance"))
   .settings(
-    version := "0.1",
+    version := persistanceVersion,
     Test / fork := true,
     commands += WithDebugCommand.withDebug,
     frgaalJavaCompilerSetting,
     annotationProcSetting,
+    javadocSettings,
+    publish / skip := false,
+    autoScalaLibrary := false,
```

Comment:
Ideally refactor those to avoid dry

Reply:
The next step I plan to do is to `publishM2` of `runtime/parser` and/or `runtime/compiler`. Publishing those will be different, as they still contain a lot of Scala code - e.g. `autoScalaLibrary` will remain `true`. We'll see whether some pattern emerges when there are more published projects...

Reply:
In general sbt's preference is to publish to Ivy repository (`publishLocal`) rather than Maven. But I guess you need the latter for other things.